5.5/Dockerfile.rhel7

```diff
@@ -13,17 +13,18 @@ FROM rhel7.2
 ENV MYSQL_VERSION=5.5 \
     HOME=/var/lib/mysql
 
-LABEL io.k8s.description="MySQL is a multi-user, multi-threaded SQL database server" \
+LABEL summary="MySQL is a multi-user, multi-threaded SQL database server" \
+      io.k8s.description="MySQL is a multi-user, multi-threaded SQL database server" \
       io.k8s.display-name="MySQL 5.5" \
       io.openshift.expose-services="3306:mysql" \
       io.openshift.tags="database,mysql,mysql55"
 
 # Labels consumed by Red Hat build service
-LABEL BZComponent="mysql55-docker" \
-      Name="openshift3/mysql-55-rhel7" \
-      Version="5.5" \
-      Release="1" \
-      Architecture="x86_64"
+LABEL com.redhat.component="mysql55-docker" \
+      name="openshift3/mysql-55-rhel7" \
+      version="5.5" \
+      release="1" \
```

Comment:
The proposal says that release label should be "The build of the image, `$IMGBUILD`". Shouldn't this be managed from outside? Will we have to bump this with every new release?